crates/task-impls/src/vote.rs

```diff
@@ -73,6 +73,7 @@ impl<
         event_stream: &Sender<HotShotEvent<TYPES>>,
     ) -> Option<HotShotTaskCompleted> {
         if vote.get_leader(&self.membership) != self.public_key {
+            error!("Received vote for a view in which we were not the leader.");
```

Comment:
I don't think this should be `error!`, but I'm just matching the other `error!` in this function (which I also don't think should be `error!`).

It might be nice if we had some rules for when to `error!`, `info!`, or `debug!`.